smack-integration-test/src/main/java/org/jivesoftware/smackx/muc/AbstractMultiUserChatIntegrationTest.java

```diff
@@ -124,16 +122,26 @@ static void createMembersOnlyMuc(MultiUserChat muc, Resourcepart resourceName) t
         }
     }
 
-    static void createModeratedMuc(MultiUserChat muc, Resourcepart resourceName) throws
-            SmackException.NoResponseException, XMPPException.XMPPErrorException,
-            InterruptedException, MultiUserChatException.MucAlreadyJoinedException,
-            SmackException.NotConnectedException,
-            MultiUserChatException.MissingMucCreationAcknowledgeException,
-            MultiUserChatException.NotAMucServiceException {
-        muc.create(resourceName);
-        Form configForm = muc.getConfigurationForm();
-        FillableForm answerForm = configForm.getFillableForm();
-        answerForm.setAnswer("muc#roomconfig_moderatedroom", true); //TODO Add this to the MucConfigFormManager?
-        muc.sendConfigurationForm(answerForm);
+    static void createModeratedMuc(MultiUserChat muc, Resourcepart resourceName)
+                    throws SmackException.NoResponseException, XMPPException.XMPPErrorException, InterruptedException,
+                    MultiUserChatException.MucAlreadyJoinedException, SmackException.NotConnectedException,
+                    MultiUserChatException.MissingMucCreationAcknowledgeException,
+                    MultiUserChatException.NotAMucServiceException,
+                    MultiUserChatException.MucConfigurationNotSupportedException {
+        MultiUserChat.MucCreateConfigFormHandle handle = muc.create(resourceName);
+        if (handle != null) {
```

Comment:
Is there a reason why we check here for nullness? Is this an error? Or can it be ignored?